src/components/subscriptions/AddUsersDropdown.jsx

```diff
@@ -1,30 +1,35 @@
 import React from 'react';
 import { Dropdown } from '@edx/paragon';
+import PropTypes from 'prop-types';
 
-export default function AddUsersDropdown() {
-  return (
-    <Dropdown className="add-users-dropdown">
-      <Dropdown.Button className="btn-primary">
-        Add users
-      </Dropdown.Button>
-      <Dropdown.Menu>
-        <Dropdown.Item
-          type="button"
-          className="btn btn-link"
-          // eslint-disable-next-line no-console
-          onClick={() => console.log('add individual user modal')}
-        >
-          With individual email address
-        </Dropdown.Item>
-        <Dropdown.Item
-          type="button"
-          className="btn btn-link"
-          // eslint-disable-next-line no-console
-          onClick={() => console.log('add bulks users modal')}
-        >
-          Import users from CSV
-        </Dropdown.Item>
-      </Dropdown.Menu>
-    </Dropdown>
-  );
-}
+import AddUserButton from './AddUsersButton';
+
+const AddUsersDropdown = ({ onSubscriptionSuccess }) => (
+  <Dropdown className="add-users-dropdown">
+    <Dropdown.Button className="btn-primary">
+      Add users
+    </Dropdown.Button>
+    <Dropdown.Menu>
+      <Dropdown.Item
+        type="button"
+        className="btn btn-link"
+      >
+        <AddUserButton onSuccess={() => onSubscriptionSuccess('Successfully assigned license(s)')} />
+      </Dropdown.Item>
+      <Dropdown.Item
+        type="button"
+        className="btn btn-link"
+        // eslint-disable-next-line no-console
+        onClick={() => console.log('add bulks users modal')}
+      >
+        Import users from CSV
```

Comment:
Isn't this already present on modal? Isn't this same as `Upload Email Addresses`?